packages/core/base/src/utils/ui.ts

```diff
@@ -7,41 +7,22 @@ export const getEnvironmentBaseUrl = (environment = ""): string => {
     return environment;
 };
 
-function blockchainTypeToLocatorStr(chain: string) {
-    switch (chain) {
-        case "solana":
-            return "sol";
-        case "ethereum":
-            return "eth";
-        case "polygon":
-            return "poly";
-        case "cardano":
-            return "ada";
-        case "bsc":
-            return "bsc";
-        default:
-            throw new Error(`Invalid chain type ${chain}`);
+function getNFTLocator(nft: NFT): string {
```

Comment:
remove :string, doesn't typescript infer that?

Reply:
done